src/theme/css/wiki-themes/discord-messages.css

```
@@ -2,15 +2,27 @@
     font-size: 16px !important;
 }
 
+.discord-message a {
+  color: #00aff4 !important;
+}
+
 .discord-messages {
     border-radius: 10px;
     background-color: #2e3035;
+    box-shadow: 7px 7px 15px rgb(66 68 73 / 50%);
+}
+
+@media only screen and (max-width: 768px) {
+.discord-message .discord-message-timestamp {
+  font-size: 10px;
+}
```

Missing and inconsistent indents
```suggestion
    .discord-message .discord-message-timestamp {
        font-size: 10px;
    }
```